docs/manual/howtos/waitset.rst

```diff
@@ -12,36 +12,38 @@
 Waitsets
 ========
 
-Waitsets are a tool in CycloneDDS to hold execution of the program until a specific condition is reached, or an amount of time has expired.
-Waitsets and listeners cover two different requirement sets.
-Whereas Listeners can only react to changes in state, a Waitset also allows the code to react when nothing changes.
-On the other hand, Listeners allow instant callbacks to specific changes.
+The Waitsets tool in CycloneDDS holds execution of the program until a specific condition is reached, or an amount of time has expired.
+Waitsets and listeners have two different requirement sets:
+
+- A Waitset  allows the code to react when nothing changes.
+- Listeners can only react to changes in state. Listeners allow instant callbacks to specific changes. 
+
 A waitset uses a StatusCondition linked to an Entity to signal to the waitset that its wait is finished.
 The StatusCondition has a list of enabled statuses, which describe the changes in status which should trigger the waitset to finish its wait.
 When creating a StatusCondition, the enabled statuses should match the type of entity it is attached to.
 Different StatusConditions linked to different Entities can be attached to a Waitset through the `attach_condition` function, and detached through the `detach_condition` function.
-After having created a Waitset with StatusConditions attached, a wait can be triggered for the Duration specified, and triggering the wait will cause one of two things to happen:
+After creating a Waitset with StatusConditions attached, a wait can be triggered for the Duration specified. Triggering the wait causes one of two things to happen:
```

Comment:
Reading over this, I am not satisfied with my original choice of phrasing this, as this implies that triggering the wait immediately gives a result

Reply:
I'm happy to wait for your updated text.